cmd/horcrux/cmd/cosigner.go

```diff
@@ -196,7 +201,10 @@ func StartCosignerCmd() *cobra.Command {
 				Threshold:   uint8(cfg.CosignerThreshold),
 			}
 
-			localCosigner := signer.NewLocalCosigner(localCosignerConfig)
+			// Initiase the localsigner (ThresholdEdSignature) of choice.
+			localsigner := signer.NewLocalSigner(cfg.ThresholdSigner, localCosignerConfig)
```

Comment:
We can hardcode this to `signer.SignerTypeSoftware` for now (constant from other suggestion).

Reply:
I believe it's resolved.